packages/importer/src/huly/unified.ts

```diff
@@ -506,13 +574,27 @@ export class UnifiedFormatImporter {
     }
   }
 
-  private processComments (comments: UnifiedComment[] = []): ImportComment[] {
-    return comments.map((comment) => {
-      return {
-        text: comment.text,
-        author: this.findAccountByEmail(comment.author)
-      }
-    })
+  private processComments (currentPath: string, comments: UnifiedComment[] = []): Promise<ImportComment[]> {
+    return Promise.all(
+      comments.map(async (comment) => {
+        const attachments: ImportAttachment[] = []
+        if (comment.attachments !== undefined) {
+          for (const attachmentPath of comment.attachments) {
+            const fullPath = path.resolve(currentPath, attachmentPath)
+            const attachmentMeta = this.fileMetadataByPath.get(fullPath)
+            if (attachmentMeta !== undefined) {
+              const importAttachment = await this.processAttachment(attachmentMeta)
```

Comment:
you can try to increase performance by processing attachments in parallel

Reply:
I've thought about this optimization too, but from my perspective, it seems like a premature optimization for a couple of reasons:

- The initialization time feels exactly the same
- Importing is a pretty rare occurrence

Also, the current approach keeps our logs clear and easy to understand.